pkg/minikube/cni/flannel.yaml

```diff
@@ -130,8 +130,8 @@ spec:
       serviceAccountName: flannel
       initContainers:
       - name: install-cni-plugin
-        #image: flannelcni/flannel-cni-plugin:v1.1.2 #for ppc64le and mips64le (dockerhub limitations may apply)
-        image: docker.io/rancher/mirrored-flannelcni-flannel-cni-plugin:v1.1.2
+        image: docker.io/flannel/flannel-cni-plugin:v1.1.2
+        #image: docker.io/rancher/mirrored-flannelcni-flannel-cni-plugin:v1.1.2
```

Comment:
why keeping the old one in comment ? we could remove the un-used deprecated image

Reply:
I believe it's commented in the upstream file, seems like a backup if the main one is down